src/Redux/actionCreators/UiContentActions.js

```diff
@@ -0,0 +1,58 @@
+// import { API } from "aws-amplify";
+// import { APIEndpoints, APIPaths } from "../../config/APIEndpoints";
```

Comment:
delete the commented lines

Reply:
this we will be needing when the api is ready, thats y didnt deleted them